common/rdm/ResponderPersonality.cpp

```diff
@@ -40,13 +40,21 @@ using std::string;
 Personality::Personality(uint16_t footprint, const string &description)
     : m_footprint(footprint),
       m_description(description) {
+        // STATIC_ASSERT(description.length() <= 32);
```

Comment:
I'm assuming this line doesn't work at all, in which case can we remove it please?

Reply:
Done.